t/summary.t

```diff
@@ -0,0 +1,131 @@
+use strict;
+use warnings;
+
+use Test::More;
+use Test::Deep;
+
+use Rex::Config;
+use Rex::Commands;
+use Rex::Commands::Run;
+use Rex::Transaction;
+
+$::QUIET = 1;
+
+subtest "distributor => 'Base'" => sub {
+
+  subtest 'exec_autodie => 0' => sub {
+    Rex::Config->set_exec_autodie(0);
+    Rex::Config->set_distributor('Base');
+    test_summary(
+      task0 => { server => '<local>', task => 'task0', exit_code => 1 },
+      task1 => { server => '<local>', task => 'task1', exit_code => 0 },
+      task2 => { server => '<local>', task => 'task2', exit_code => 0 },
+      task3 => { server => '<local>', task => 'task3', exit_code => 1 },
+      task4 => { server => '<local>', task => 'task4', exit_code => 1 },
+    );
+  };
+
+  subtest 'exec_autodie => 1' => sub {
+    Rex::Config->set_exec_autodie(1);
+    Rex::Config->set_distributor('Base');
+    test_summary(
+      task0 => { server => '<local>', task => 'task0', exit_code => 1 },
+      task1 => { server => '<local>', task => 'task1', exit_code => 2 },
+      task2 => { server => '<local>', task => 'task2', exit_code => 0 },
+      task3 => { server => '<local>', task => 'task3', exit_code => 1 },
+      task4 => { server => '<local>', task => 'task4', exit_code => 1 },
+    );
+  };
+};
+
+subtest "distributor => 'Parallel_ForkManager'" => sub {
```

Comment:
These tests will die if `Parallel::ForkManager` is not present. Please run them only if the module is installed (and adjust the test plan accordingly.